lib/lib.es2015.core.d.ts

```diff
@@ -37,7 +37,7 @@ interface Array<T> {
       * @param thisArg If provided, it will be used as the this value for each invocation of
       * predicate. If it is not provided, undefined is used instead.
       */
-    findIndex(predicate: (value: T) => boolean, thisArg?: any): number;

```

Comment:
these files are generated from the build and should not be changed manually. so please remove these.

see https://github.com/Microsoft/TypeScript/blob/master/CONTRIBUTING.md#contributing-libdts-fixes for more details.